rpcs3/Emu/Cell/SPUDisAsm.h

```diff
@@ -148,11 +148,11 @@ class SPUDisAsm final : public PPCDisAsm
 	using field_de_t = decltype(spu_opcode_t{}.de);
 	void DisAsm(std::string op, field_de_t de, const char* a1)
 	{
-		Write(fmt::format("%s%s %s", FixOp(op).c_str(), BrIndirectSuffix(de), a1));
+		Write(fmt::format("%s %s", FixOp(op.append(BrIndirectSuffix(de))).c_str(), a1));
```

Comment:
fmt::format does not need c_str for eternity...
Why append?

Reply:
It doesn't need c_str, it's for consistency with other FixOp() uses.